tests/Unit/NumericalAlgorithms/DiscontinuousGalerkin/Actions/Test_ImposeBoundaryConditions.cpp

```diff
@@ -38,7 +39,7 @@ struct PrimitiveVar : db::SimpleTag {
   using type = Scalar<DataVector>;
 };
 
-struct BoundaryCondition {
+struct BoundaryCondition : MarkAsAnalyticSolution {
```

Comment:
Currently we only support dirichlet boundary conditions imposed by an analytic solution, and since that's being checked in `ImposeBoundaryConditions.hpp` I made this change in the test. You don't need to change anything in `src`. I added a comment